### ISSUE TITLE
install broken with PR 1744

### ISSUE THREAD

**Issue by abarz722** [CactuseSecurity/firewall-orchestrator#1758]
on my debian machine I get now:

TASK [middleware : Install webserver packages] **********************************************************************************************************************************************
ok: [fworch-srv] => (item=apache2)
failed: [fworch-srv] (item=libapache2-mod-wsgi) => changed=false
  ansible_loop_var: item
  item: libapache2-mod-wsgi
  msg: No package matching 'libapache2-mod-wsgi' is available

**Comment by tpurschke** [comment]
yep - already on it - debian11, I assume? Fix will be there in a couple hours
